### PR TITLE
Add GCI task series "Use coala"

### PR DESCRIPTION
**Pull request by jayvdb**
The markdown comes from the following three pages:

- https://github.com/coala/coala/wiki/Google-Code-In-Task-Use-coala
- https://github.com/coala/coala/wiki/Google-Code-In-Task-Use-coala-Advanced
- https://github.com/coala/coala/wiki/Google-Code-In-Task-Implement-coala

Related to https://github.com/coala/projects/issues/299

### CODE REVIEW

**Comment by jayvdb** [comment]
the stackoverflow tags part looks like it should be a feature of the core app

File: data/projects.js [hunk 1]
```diff
@@ -680,5 +680,57 @@ coala should support generating metrics for your code.",
...
+        ],
+        "difficulty": "medium",
+        "mentors" : [
+            "jayvdb",
+            "sils"
+        ],
+        "initiatives": ["GCI"],
+        "tags": ["community", "docker"],
+        "markdown": "use_coala_3.md",
+        "collaborating_projects": ["docker-coala-base"]
```
Comment (meetmangukiya) [inline data/projects.js]: oh boy, we need to split this file :worried:

Comment (hemangsk) [inline data/projects.js]: yea it is in my phase - II action plan ;)

**Comment by yukiisbored** [comment]
ack https://github.com/coala/projects/commit/de3ee7907f1ac9c56e4c8495073cc1d431504817

**Comment by yukiisbored** [comment]
LGTM, we may need to improve the UI to make it more friendly for GCI bite sized tasks.

**Comment by jayvdb** [comment]
@rultor merge

**Comment by jayvdb** [comment]
reack ad51afe

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**